### ISSUE TITLE
Needles on ND have disappeared

### ISSUE THREAD

**Issue by gillesvana** [flybywiresim/aircraft#8438]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-01-20T23:45:41+00:00",
    "ref": "refs/heads/master",
    "sha": "d70bd6c8d838aaa41319596ce13ee7f7ede5c1c9",
    "actor": "frankkopp",
    "event_name": "manual",
    "pretty_release_name": "master:d70bd6c8",
    "version": "v0.12.0-dev.d70bd6c"
}
```

### Describe the bug

no more NDB needles displayed on the ND

### Expected behavior

information of nd well write  in left down corner ND, but no needle

### Steps to reproduce

near LFBD for BD adf (ndb)

**Comment by BokBoy** [comment]
This issue was reported to discord once and is a known issue, as far as I know.

**Comment by Watsi01** [comment]
Affirmative, known issue. But thanks for reporting.

**Comment by tracernz** [comment]
Working fine for me. Navdata issue? Try one of the default aircraft to see.
![image](https://github.com/flybywiresim/aircraft/assets/9995998/cb421b93-1841-4841-8b29-13c4e88aeea8)

**Comment by tracernz** [comment]
BD also working fine.
![image](https://github.com/flybywiresim/aircraft/assets/9995998/4e3aa900-1366-45c8-b579-b9ffa850b157)

**Comment by gillesvana** [comment]
Thanks for your reply. Why 2 persons have writed to me that this issues was known? Envoyé depuis mon téléphone-------- Message original --------De : Michael Corcoran ***@***.***>Date : jeu. 25 janv. 2024 à 08:19À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Author ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
BD also working fine.
image.png (view on web)

**Comment by gillesvana** [comment]
Do you use the same last dev release? Envoyé depuis mon téléphone-------- Message original --------De : Michael Corcoran ***@***.***>Date : jeu. 25 janv. 2024 à 08:15À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Author ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
Working fine for me. Navdata issue? Try one of the default aircraft to see.
image.png (view on web)

**Comment by tracernz** [comment]
I use the latest, yes ([cf1e4f4](https://github.com/flybywiresim/aircraft/commit/cf1e4f4a3cebd96c68cdc4e82f6a1a6d875403ef)).

**Comment by BokBoy** [comment]
> Thanks for your reply. Why 2 persons have writed to me that this issues was known?

I sent it first, wasn't sure but the other person confirmed it.

**Comment by gillesvana** [comment]
Yes I do! Envoyé depuis mon téléphone-------- Message original --------De : RogePete ***@***.***>Date : jeu. 25 janv. 2024 à 09:15À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Mention ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
@gillesvana: We have indeed different findings. Do you use Navigraph navdata for the sim?

**Comment by tracernz** [comment]
Are you airborne above the NDB when you're trying to receive it? The MSFS NDB simulation rarely allows you to receive NDB signals on the ground.

**Comment by gillesvana** [comment]
Yes of course cruise 5000 feet around nb, Ne, nb all without needle. I have tested with fenix and it's good. Nav data updated with navigraph. Envoyé depuis mon téléphone-------- Message original --------De : Michael Corcoran ***@***.***>Date : jeu. 25 janv. 2024 à 22:27À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Mention ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
Are you airborne above the NDB when you're trying to receive it? The MSFS NDB simulation rarely allows you to receive NDB signals on the ground.

**Comment by gillesvana** [comment]
No needle Adf at home... But Vor OK. Here tested all NDB around LFBD. No needle! Envoyé depuis mon téléphone-------- Message original --------De : Michael Corcoran ***@***.***>Date : jeu. 25 janv. 2024 à 08:19À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Author ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
BD also working fine.
image.png (view on web)

**Comment by gillesvana** [comment]
Here Marseille, LFML no NDB needle. And normally with or without frequency receive ground or not you must have the needle, if not active she is at horizontally and right position.... With no flag information on left down corner. I have the last of today release. Regards. Envoyé depuis mon téléphone-------- Message original --------De : Michael Corcoran ***@***.***>Date : jeu. 25 janv. 2024 à 22:27À : flybywiresim/aircraft ***@***.***>Cc : gillesvana ***@***.***>, Mention ***@***.***>Objet : Re: [flybywiresim/aircraft] Needles on ND have disappeared (Issue #8438)
Are you airborne above the NDB when you're trying to receive it? The MSFS NDB simulation rarely allows you to receive NDB signals on the ground.

**Comment by Watsi01** [comment]
I tried it again in EDDF with the according NDB (297.0) and the A32NX. Not shown in ND.

**Comment by Watsi01** [comment]
We have found a first clue (why some of us can see the needles and some can't). If you spawn at the gate, cold&dark and fire the plane up within a normal flight, you won't see any needles. If you spawn on the runway (with the aircraft running) you will see needles. 
Now we have to find out what the problem is.
